package.json

```diff
@@ -866,6 +866,7 @@
         "@microsoft/vscode-azext-github": "^1.0.2",
         "@microsoft/vscode-azext-utils": "^3.1.4",
         "@microsoft/vscode-azureresources-api": "^2.0.2",
+        "@swc/core-linux-x64-gnu": "^1.12.9",
```

Comment:
Probably need to move this into the azure-pipelines since we can't run npm install with this